src/styles/blocks/card.scss

```diff
@@ -0,0 +1,117 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  transition: transform 0.3s ease;
+
+  &__image {
+    display: block;
+    margin: 0 auto;
+  }
+
+  &__title {
+    margin: 40px 0 0;
```

Comment:
You can reset default margin for p tag once instead of resetting each time and use margint-top: 40px; - it's more readable
p {
  margin: 0;
}